boards.txt

```diff
@@ -18494,15 +18494,9 @@ m5stack_core.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
 m5stack_core.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
 m5stack_core.menu.PartitionScheme.fatflash.build.partitions=ffat
 m5stack_core.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
-m5stack_core.menu.PartitionScheme.rainmaker=RainMaker 4MB
+m5stack_core.menu.PartitionScheme.rainmaker=RainMaker
 m5stack_core.menu.PartitionScheme.rainmaker.build.partitions=rainmaker
-m5stack_core.menu.PartitionScheme.rainmaker.upload.maximum_size=1966080
-m5stack_core.menu.PartitionScheme.rainmaker_4MB=RainMaker 4MB No OTA
-m5stack_core.menu.PartitionScheme.rainmaker_4MB.build.partitions=rainmaker_4MB_no_ota
-m5stack_core.menu.PartitionScheme.rainmaker_4MB.upload.maximum_size=4038656
-m5stack_core.menu.PartitionScheme.rainmaker_8MB=RainMaker 8MB
-m5stack_core.menu.PartitionScheme.rainmaker_8MB.build.partitions=rainmaker_8MB
-m5stack_core.menu.PartitionScheme.rainmaker_8MB.upload.maximum_size=4116480
+m5stack_core.menu.PartitionScheme.rainmaker.upload.maximum_size=3145728
```

Comment:
@Tinyu-Zhao Why are you rollbacking the RainMaker partitions? It got fixed recently in 3.0.4.
As RainMaker partition uses 2 OTA partitions, the maximum upload size should be 1966080, not 3145728.
The binary will flash fine, but the board will stay in boot loop.

Reply:
Thanks for the tip, I just pushed the update.